.expeditor/scripts/release_habitat/update_api_docs.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+component=sup
+
+# shellcheck source=.expeditor/scripts/release_habitat/shared.sh
+source .expeditor/scripts/release_habitat/shared.sh
+
+echo "--- :hammer_and_pick: Generating Habitat Supervisor API docs"
+
+#  Check if hub is installed and install if not
+hub_check=$(which hub)
+if [ -z "$hub_check" ]; then
+  install_hub
+fi
+
+# Check if node is installed and install if not
+node_version=$(node -v)
+if [ -z "$node_version" ]; then
+  sudo apt-get install -y nodejs
+fi
```

Comment:
You could replace lines 19-22 with `hab pkg install core/node --binlink`

Reply:
Agreed - will update.